### ISSUE TITLE
BCNM Royal Jelly - The mobs would not die

### ISSUE THREAD

**Issue by Rivver91** [EdenServer/community#1974]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

Just did BCNM 40 Royal Jelly. The first jelly we fought would not die. It would take damage, but was stuck at 1% hp. This was NOT the queen jelly. It was the first mob, well before it ever got to the center.

**Comment by spacewolf007** [comment]
This is Spacewolf. I was a part of this fight and attempted to submit a ticket but do not see it here nor on discord. The first mob on the left stopped at 1% HP and would not die no matter the damage. I used the sky orb on this bcnm so feel free to contact me regarding this issue. 

Thank you

**Comment by Rivver91** [comment]
I subsequently completed this bcnm 3 times, but other players have also had issues such as the jellies not combining to form the queen jelly.

**Comment by godmode** [comment]
Its helpful to provide reproduceable steps or this may take much longer to fix. What did you do during the fight that this happened? Approximate time elapsed between and during kills, was one casting, etc.

**Comment by Rivver91** [comment]
I wish I could provide reproduceable steps, but I've only experienced the issue once. The only thing I can think of is that I switched targets right as the jelly was going to die (one of the other players was still attacking it).

**Comment by GalkaZajin** [comment]
Were you using holy bolts by any chance?

**Comment by Rivver91** [comment]
No, no holy bolts.

**Comment by slassen** [comment]
Not enough information on this to reproduce, considering this issue resolved. Please add more details or comment if it happens again.

**Comment by Namiea012** [comment]
Hello everyone,

Unfortunately, this exact same issue happened to me and my party on 8/4/20 at about 2:45 AM EST. 

We successfully completed this 3 times on 8/4/20 and 7 times on 8/3/20 without issue. To the best of my knowledge to reproduce this error, the last jelly has to get to 1% health just before wanting to combine or change into the Queen. The last jelly mob gets stuck at 1% and you have no choice but to wait until the timer runs out and you get kicked out.

The last jelly appears to "take" damage (with numbers being logged) but its health doesn't actually go down from 1%. 

I'm not sure how helpful these screenshots are but this shows that the mob is clearly at 1% for a few minutes and is just stuck.

![Namiea_2020 08 04_024647](https://user-images.githubusercontent.com/67996039/89263143-b5662080-d5fe-11ea-8c90-b9d594081741.png)

![Namiea_2020 08 04_025030](https://user-images.githubusercontent.com/67996039/89263149-b7c87a80-d5fe-11ea-9630-3d1bdd57d841.png)

I also submitted a Help Desk request. 

Thank you so much!
